build.gradle

```diff
@@ -31,3 +31,13 @@ else {
 	throw new GradleException("GHIDRA_INSTALL_DIR is not defined!")
 }
 //----------------------END "DO NOT MODIFY" SECTION-------------------------------
+
+def sleigh = new File(ghidraInstallDir).getCanonicalPath() + "/support/sleigh.bat"
```

Comment:
Unfortunately this won't work on an OS other than Windows... To compile the sleigh files you use `ant` in the `data` directory (which build.xml is).